component/pom.xml

```diff
@@ -186,9 +214,265 @@
                                 <PORT>${siddhi-cdc-mysql.port}</PORT>
                                 <DOCKER_HOST_IP>${docker.container.siddhi-cdc-mysql.ip}</DOCKER_HOST_IP>
                             </environmentVariables>
-                            <suiteXmlFiles>
-                                <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
-                            </suiteXmlFiles>
+                            <test>
+                                org.wso2.extension.siddhi.io.cdc.source.TestCaseOfMySQLCDC
```

Comment:
Why is there different test cases for each type of database? Isn't it enough to parameterise and reuse the same test cases?

Reply:
Fixed with commit: 3d57598